src/components/bookmark-button/icons/index.tsx

```diff
@@ -0,0 +1,56 @@
+import s from './icons.module.css'
+
+/**
+ * We need these icon to be an inline svg so we can manipulate the colors of the
+ * 'plus' & 'minus' sign lines based in the varying bookmark hover states
```

Comment:
This _should_ be possible by setting `color` for the `<button>` the icons are placed in. This is something that works for me with a quick test:

```
<button style={{ color: 'red' }}>
  <IconBookmarkAdd16 />
</button>
```

<img width="141" alt="CleanShot 2022-08-15 at 18 58 25@2x" src="https://user-images.githubusercontent.com/43934258/184734741-8a74067d-2c63-43fd-8c26-62849c6a15e1.png">

Reply:
It's not that I need to override the color, it's that I need to conditionally show the 'plus' or 'minus' icon based on the hover state. I'm definitely open to a different way of doing this! lmk if you have ideas.

Reply:
I need to have a way to target just the 'plus' or just the 'minus' icon in order to show / hide them on hover and I'm not sure how to do that with just the flight icon unless I'm targeting elements, which feels prone to error later on if the markup of the icon shifts.

Reply:
Ahhh I see now! Maybe rendering two icons based on the state and then hiding one with CSS will work?

```tsx
// If tutorial is not bookmarked
<button className={s.bookmarkButton}>
  <IconBookmark16 className={s.defaultStateIcon} />
  <IconBookmarkAdd16 className={s.hoverStateIcon} />
</button>
```

```tsx
// If tutorial is bookmarked
<button className={s.bookmarkButton}>
  <IconBookmarkFill16 className={s.defaultStateIcon} />
  <IconBookmarkRemoveFill16 className={s.hoverStateIcon} />
</button>
```

```css
.bookmarkButton {
  color: var(--token-color-foreground-faint);

  &:hover, &:focus {
    color: var(--token-color-foreground-strong);

    // Hide in hover state
    & .defaultStateIcon {
      display: none;
    }

    // Show in hover state
    & .hoverStateIcon {
      display: block;
    }
  }
}

// Show in default state
.defaultStateIcon {
  display: block;
}

// Hide in default state
.hoverStateIcon {
  display: none;
}
```

Reply:
That's smart 😄 I'll try that out!

Reply:
Made that update! thanks for the idea 😸